libraries/Microsoft.Bot.Builder.AI.LanguageGeneration/ThrowingErrorListener.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Antlr4.Runtime;
+using Antlr4.Runtime.Misc;
+
+namespace Microsoft.Bot.Builder.AI.LanguageGeneration
+{
+    public class ThrowingErrorListener : BaseErrorListener
+    {
+        public static ThrowingErrorListener INSTANCE { get; } = new ThrowingErrorListener();
```

Comment:
A a minimum, please comply with the coding style throughout this codebase. ALL CAPS variable names, for example, don't comply.

Reply:
ok